govcd/vapp_test.go

```diff
@@ -1490,3 +1490,59 @@ func createVappForTest(vcd *TestVCD, vappName string) (*VApp, error) {
 
 	return vapp, nil
 }
+
+// Test_AddNewVMFromMultiVmTemplate creates VM from OVA holding a few VMs
+func (vcd *TestVCD) Test_AddNewVMFromMultiVmTemplate(check *C) {
+
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vapp was not successfully created at setup")
+	}
+
+	// Find VApp
+	if vcd.vapp.VApp == nil {
```

Comment:
Why do we skip when no vApp was found if we are creating a vApp later in the test?

Reply:
removed